tracer/src/Datadog.Trace/Agent/Transports/ApiWebRequest.cs

```diff
@@ -8,13 +8,22 @@
 using System.Text;
 using System.Threading.Tasks;
 using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
 
 namespace Datadog.Trace.Agent.Transports
 {
-    internal class ApiWebRequest : IApiRequest
+    internal class ApiWebRequest : IApiRequest, IMultipartApiRequest
     {
+        private const string Boundary = "faa0a896-8bc8-48f3-b46d-016f2b15a884";
```

Comment:
Just wondering why you switched this to a `Guid`? Is it worth keeping some of the `-------`, as that's pretty standard (and makes the body visually easier to parse?) not bothered really, just wondered :)

Reply:
Discussed offline: 

> I just ran a test with the HttpClient implementation that has the multipart built in and copied the boundary from that sample and created a constant from that
> The value must be valid, because comes from the framework implementation